Source/monster.cpp

```diff
@@ -4829,13 +4829,13 @@ void TalktoMonster(Monster &monster)
 	}
 }
 
-void SpawnGolem(int i, Point position, Missile &missile)
+void SpawnGolem(int monsterId, Point position, Missile &missile)

```

Comment:
Should be maybe call this `id` since it's ambiguous if it's the player or monster id?

Reply:
Fixed